### ISSUE TITLE
Any solution to add html property to src (image) property ?

### ISSUE THREAD

**Issue by mijatovic85** [dimsemenov/PhotoSwipe#1340]
In documentation I found Additional important notes:

To avoid conflicts with third-party modules, slide that has html property, should not have src (image) property. Is there any solution to add html property to src (image) property ?

**Comment by dimsemenov** [comment]
I don't entirely understand the question. This warning does not prevent you from passing `<img>` element  via `html` property, it just can not be zoomed.

**Comment by mijatovic85** [comment]
Please see screenshot - http://prntscr.com/f4qug1 , I want to archive this thumbnail slide, practically one slide is image and caption or `html` property. Is this possible to archive? Thank you

**Comment by dimsemenov** [comment]
You may pass any HTML content via `html` property and style it however you wish.

**Comment by mijatovic85** [comment]
Yes, thank you, I know that functionality, but I am not sure can I slide thumbnails, or first I need to click on the image , and then I will get functionality of sliding another slides?

**Comment by mijatovic85** [comment]
@dimsemenov , I have a question for you as the author of this plugin, please answer me. 
is that possible to achieve PhotoSwipe gallery with thumbnails like this http://prntscr.com/f5e272 ?

Practically , slide images without click on the image , as you can see on the screenshot
